src/dotnet/APIView/APIViewWeb/DTOs/CommentUpdatesDto.cs

```diff
@@ -0,0 +1,43 @@
+using System.Text.Json.Serialization;
+using APIViewWeb.LeanModels;
+
+namespace APIViewWeb.DTOs
+{
+    public enum CommentThreadUpdateAction
+    {
+        CommentCreated = 0,
+        CommentTextUpdate,
+        CommentResolved,
+        CommentUnResolved,
+        CommentUpVoteToggled,
+        CommentDeleted
```

Comment:
Do we not allow to lock the comment to make it resolvable by reviewer only? Or is it only possible when comment is created for the first time? Do we need these many options or just 3 Enums such as Created, Edited, Deleted?